ulwgl_run.py

```diff
@@ -16,15 +17,15 @@
 def parse_args() -> Union[Namespace, Tuple[str, List[str]]]:  # noqa: D103
     opt_args: Set[str] = {"--help", "-h", "--config"}
     exe: str = Path(__file__).name
-    usage: str = f"""
-example usage:
-  WINEPREFIX= GAMEID= PROTONPATH= {exe} /home/foo/example.exe
-  WINEPREFIX= GAMEID= PROTONPATH= {exe} /home/foo/example.exe -opengl
-  WINEPREFIX= GAMEID= PROTONPATH= {exe} ""
-  WINEPREFIX= GAMEID= PROTONPATH= PROTON_VERB= {exe} /home/foo/example.exe
-  WINEPREFIX= GAMEID= PROTONPATH= STORE= {exe} /home/foo/example.exe
-  {exe} --config /home/foo/example.toml
-    """
+    usage: str = (
+        "example usage:\n"
+        f"  WINEPREFIX= GAMEID= PROTONPATH= {exe} /home/foo/example.exe\n"
+        f"  WINEPREFIX= GAMEID= PROTONPATH= {exe} /home/foo/example.exe -opengl\n"
+        f"  WINEPREFIX= GAMEID= PROTONPATH= {exe} \"\"\n"
+        f"  WINEPREFIX= GAMEID= PROTONPATH= PROTON_VERB= {exe} /home/foo/example.exe\n"
+        f"  WINEPREFIX= GAMEID= PROTONPATH= STORE= {exe} /home/foo/example.exe\n"
+        f"  {exe} --config /home/foo/example.toml\n"
+    )
     parser: ArgumentParser = argparse.ArgumentParser(
```

Comment:
I'd really like to automate style changes like this via a formatter (e.g. ruff format) then have a bot just format the files on every push. IIRC, ruff format can be configured to set a line-length. That way, we can just focus on the logic and developing core functionality.

Aside from this, personally, I think having those newline characters added makes the string look worse...

Reply:
> then have a bot just format the files on every push. IIRC, ruff format can be configured to set a line-length.

I understand that, but the bot will fail sometimes. It might not split at the newline, which is the more convenient place to split. I do agree though that consistent formatting it key.

> Aside from this, personally, I think having those newline characters added makes the string look worse...

IMO it is a tradeoff, you are either breaking the indentation or you get the newline characters. Personally I find breaking indentation worse, but this is not a dealbreaker in any way.

Reply:
That's fair. I will remove that commit then.